empress/__init__.py

```diff
@@ -67,31 +72,38 @@ def cluster(self, n) -> List['ReconGraphWrapper']:
         """
         pass
 
+
 class CostRegionWrapper(Drawable):
-    def __init__(self, info):
+    def __init__(self, cost_vectors, switch_min, switch_max, loss_min, loss_max):
         """
```

Comment:
Note that I have recently modified costscape so that switch_min and switch_max are called transfer_min and transfer_max.  Also, loss_min and loss_max are no longer used.  Instead, loss is fixed at 1 and we have dup_min and dup_max.  That is, the costscape plot has transfer costs relative to loss on the y-axis and duplication cost relative to loss on the x-axis.

Reply:
Note that in our usage, a "region" is an area in the costscape plot where all event costs in that region give rise to the same solutions.  So, I suggest changing compute_cost_region to compute_cost_regions.  In general, a costscape plot shows many regions.

Reply:
My bad on this. Will fix this before merging.

Reply:
If you are going to overload the constructor, you probably want to do specify the types in the function signature or else do some type-checking, e.g. to make sure someone does not pass `cost_vectors` as `info`.

Reply:
@wutron I'm not trying to overload the constructor. I used `info` as a placeholder in my draft.

Reply:
Ah, sorry, I completely misread the changes.